src/main/java/pages/opportunity/OpportunityForm.java

```diff
@@ -0,0 +1,196 @@
+package pages.opportunity;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.CacheLookup;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.Select;
+import pages.LookupPage;
+import pages.basepages.FormBase;
+
+import java.util.LinkedList;
+import java.util.Set;
+
+/**
+ * Created by Virginia Sanabria on 8/30/2015.
+ */
+public class OpportunityForm extends FormBase{
+    @FindBy(id = "opp3")
+    @CacheLookup
+    private WebElement opyNameTxt;
+
+    @FindBy(id = "opp5")
+    @CacheLookup
+    private WebElement typeComBox;
+
+    @FindBy(id = "opp6")
+    @CacheLookup
+    private WebElement leadSourceComBox;
+
+    @FindBy(id = "opp7")
+    @CacheLookup
+    private WebElement amountTxt;
+
+    @FindBy(css = "img[title *= 'Account Name Lookup']")
+    @CacheLookup
+    private WebElement lookupIcon;
+
+    @FindBy(id = "opp9")
+    @CacheLookup
+    private WebElement closeDateTxt;
+
+    @FindBy(id = "opp10")
+    @CacheLookup
+    private WebElement nextStepTxt;
+
+    @FindBy(id = "opp11")
+    @CacheLookup
+    private WebElement stageComBox;
+
+    @FindBy(id = "opp12")
+    @CacheLookup
+    private WebElement probabilityTxt;
+
+    @FindBy(xpath= "//td[contains(., 'Order Number')]/following-sibling::td/input")
```

Comment:
you are right, but using find By, it will get the first element,  that element is the input that I needed and it will work fine.
Anyway , I will change the locator: //td[contains(., 'Order Number')]/following-sibling::td[1]/input